content/hi/docs/tasks/tools/included/optional-kubectl-configs-zsh.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "zsh ऑटो-कम्पलीशन"
+description: "zsh ऑटो-कम्पलीशन के लिए कुछ वैकल्पिक कॉन्फ़िगरेशन|"
```

Comment:
```suggestion
description: "zsh ऑटो-कम्पलीशन के लिए कुछ वैकल्पिक कॉन्फ़िगरेशन।"
```

Reply:
Please note that `।` and `|` are not the same character and we should use `।` ideally.